src/components/TVOptionItem.css

```diff
@@ -0,0 +1,9 @@
+.ListIcon {
```

Comment:
Volgens de aandachtspuntenlijst is het beter om hier kleine letters te gebruiken